test/integration/ci-operator-configresolver.sh

```diff
@@ -1,65 +1,88 @@
 #!/bin/bash
 source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
 
-rsync_tmp_dir=$(mktemp -d)
 function cleanup() {
-    rm -rf $rsync_tmp_dir
     os::test::junit::reconcile_output
     os::integration::configresolver::check_log
     os::cleanup::processes
 }
 trap "cleanup" EXIT
 
 suite_dir="${OS_ROOT}/test/integration/ci-operator-configresolver/"
-cp -a "${suite_dir}"/* "${BASETMPDIR}"
-cp -a "${OS_ROOT}/test/multistage-registry" "${BASETMPDIR}"
-actual="${BASETMPDIR}/actual"
-mkdir -p "${actual}"
-expected="${suite_dir}/expected"
+unused_dir=${suite_dir}/unused
 
 os::test::junit::declare_suite_start "integration/ci-operator-configresolver"
-# This test validates the ci-operator-configresolver tool
 
-os::integration::configresolver::start "${BASETMPDIR}/configs" "${BASETMPDIR}/multistage-registry/registry" "${BASETMPDIR}/config.yaml"
+test_validation() {
```

Comment:
Why is this a function?

Reply:
maybe

```
os::test::junit::declare_suite_start "integration/ci-operator-configresolver/validation"
```

Reply:
I made it a function just to isolate the local variables from the rest of the script.

The two types of tests are different enough that I considered making a separate script, but didn't know exactly what file/directory structure to use.  WDYT?

Reply:
Do something to make it look as similar to the rest as you can, if that means a new script, sure, it's light-weight